tasks/configure-OpenBSD.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+# XXX rpc.lockd(8) of OpenBSD does not support client-side of the protocol
```

Comment:
the fact that NFS lock does not work in OpenBSD will be documented in README, then.